src/bson.ts

```diff
@@ -25,6 +27,21 @@ export {
   UUID
 } from 'bson';
 
+export type BSONElement = BSON.OnDemand['BSONElement'];
+
+export function parseToElementsToArray(bytes: Uint8Array, offset?: number): BSONElement[] {
+  const res = BSON.onDemand.parseToElements(bytes, offset);
+  return Array.isArray(res) ? res : [...res];
+}
+
+// eslint-disable-next-line @typescript-eslint/unbound-method
+export const getInt32LE = BSON.onDemand.NumberUtils.getInt32LE;
```

Comment:
Fwiw, I know https://github.com/mongodb/js-bson/pull/665 is already merged but you should be able to get around this eslint warnings in a 'clean' way by typing the methods as e.g.

```js
  getInt32LE: (this: void, source: Uint8Array, offset: number) => number;
```